www/script.js

```diff
@@ -2,22 +2,41 @@ let nodes = document.querySelectorAll('.node');
 let pipes = document.querySelectorAll('.piping');
 let completionPackageIri = 'asset:Package1';
 
-nodes.forEach((node) => {
-    node.addEventListener('click', async (event) => {
-        await handleNodeClick(node, event);
-        await updateInCommissioningPackage();
-    });
-});
-
 window.addEventListener('load', async () => {
     for (const node of nodes) {
         await makeSparqlAndUpdateStore(node.id, boundary_actions.delete, boundary_parts.boundary);
         await makeSparqlAndUpdateStore(node.id, boundary_actions.delete, boundary_parts.insideBoundary);
         node.classList.remove('insideBoundary', 'boundary');
         removeCommissionHighlight(node);
     }
+
+    for (const pipe of pipes) {
```

Comment:
Kan ikke pipes highlightes med samme metode som noder? Jeg mener, hvorfor funker ikke handleNodeClick på pipes?

Reply:
Prøvde en rask test nå med å slå i sammen nodes og pipes til en liste og så legge til en event listener her uten så mye suksess. 
Pipes og noder må håndteres forskjellig
- Når en pipe blir valgt så får den klassen boundary og får rød farge. En node vil og få klassen boundary, men vi skal ikke endre fargen til rød. 
- En pipe som boundary blir ikke avvalgt på samme måte som en vanlig node. Dette er fordi når en pipe blir valgt så legges det et nytt element over denne. Det må derfor legges til en event listener på dette nye highlight elementet. 

Det kunne nok vært mulig å ha en event listener som håndterte både elementer av klasse piping og node, men da måtte vi hatt if sjekker i denne event listneren for å håndtere de to klassene ulikt. Jeg syns derfor det blir mer tydelig å ha logikken for disse to klassene separat.